compiler/src/iree/compiler/Dialect/HAL/IR/HALOps.td

```diff
@@ -589,8 +589,8 @@ def HAL_AllocatorSelectAttrOp : HAL_PureOp<"allocator.select.attr", [
 
   let arguments = (ins
     HAL_DeviceOptimalAttr:$optimal_set,
-    HAL_MemoryTypeBitfieldAttr:$memory_types,
-    HAL_BufferUsageBitfieldAttr:$buffer_usage
+    HAL_MemoryType:$memory_types,
+    HAL_BufferUsage:$buffer_usage
```

Comment:
this op should not change (it's the attribute-based version, the non-attr op below is the value-based one - that could be updated to use the HAL_MemoryType/HAL_BufferUsage instead of I32, though)